subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/AvailableJavaHomes.java

```diff
@@ -116,25 +115,26 @@ public static Jvm getJdk(final JavaVersion version) {
      */
     public static List<Jvm> getJdks(JavaVersion... versions) {
         final Set<JavaVersion> remaining = Sets.newHashSet(versions);
-        return getAvailableJdks(element -> remaining.remove(element.getJavaVersion()));
+        return getAvailableJdks(element -> remaining.remove(element.getLangageVersion()));
```

Comment:
Totally missed that in other reviews - we have a typo there: `getLanguageVersion`